buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -390,8 +390,11 @@
       "value": "Debug",
       "allowOverride": true
     },
+    "PB_TargetQueue": {
+      "value": "Windows.10.Amd64"
+    },
     "PB_CreateHelixArguments": {
-      "value": "/t:CloudBuild /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=true\" /p:\"BuildMoniker=none\" /p:\"TargetQueue=Windows.10.Amd64\" /p:\"TestProduct=corefx\" /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Windows_NT\" /p:FilterToOSGroup=Windows_NT",
+      "value": "/t:CloudBuild /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=true\" /p:\"BuildMoniker=none\" /p:\"TargetQueue=$(PB_TargetQueue)\" /p:\"TestProduct=corefx\" /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Windows_NT\" /p:FilterToOSGroup=Windows_NT",
```

Comment:
I'm no t sure I see the value of making TargetQueue a parameter here if it's not reused plus it's only one value, so I would keep this PR about using `+` instead of commas on pipeline.json